cmd/server/main.go

```diff
@@ -62,7 +62,9 @@ func main() {
 	}
 
 	if *versionInfo {
-		version.PrintVersion()
+		if err := version.PrintVersion(); err != nil {
+			klog.ErrorS(err, "failed to print version")
+		}
```

Comment:
```suggestion
		if err := version.PrintVersion(); err != nil {
			klog.ErrorS(err, "failed to print version")
			os.Exit(1)
		}
```

we want to exit 1 in case of error printing the version